train.py

```diff
@@ -33,42 +33,40 @@ def train(model_name, seed, epochs, patience, b_size, l_rate, w_decay, warmup, m
     print(
         f'Configuration:\n model_name: {model_name}\n data_name: {data_name}\n max epochs: {epochs}\n'
         f' patience: {patience}\n seed: {seed}\n batch_size: {b_size}\n l_rate: {l_rate}\n warmup: {warmup}\n '
-        f'weight_decay: {w_decay}\n cf_hidden_dim: {cf_hidden_dim}\n resume checkpoint: {resume}\n')
+        f'weight_decay: {w_decay}\n cf_hidden_dim: {cf_hidden_dim}\n checkpoint: {checkpoint}\n')
     if model_name in ['pure_gnn', 'roberta_gnn']:
         print('GNN layer:', gnn_layer_name)
 
     pl.seed_everything(seed)
 
     # the data preprocessing
 
-    train_loader, val_loader, test_loader, additional_params = get_dataloaders(model_name, b_size, data_name)
+    train_loader, val_loader, test_loader, add_params = get_dataloaders(model_name, b_size, data_name, checkpoint)
 
     optimizer_hparams = {"lr": l_rate, "weight_decay": w_decay, "warmup": warmup, "max_iters": max_iters}
 
     model_params = {
         'model': model_name,
         'gnn_layer_name': gnn_layer_name,
         'cf_hid_dim': cf_hidden_dim,
-        'checkpoint': resume,
-        **additional_params
+        'checkpoint': checkpoint,
+        **add_params
     }
 
     trainer = initialize_trainer(epochs, patience, model_name, l_rate, w_decay, warmup, seed, data_name, transfer)
 
     # optionally resume from a checkpoint
-    if not transfer and resume is not None:
+    if not transfer and checkpoint is not None:
         print(f'=> intending to resume from checkpoint')
-        if os.path.isfile(resume):
-            print(f"=> loading checkpoint '{resume}'")
-            model = DocumentClassifier.load_from_checkpoint(resume)
-            print(f"=> loaded checkpoint '{resume}'\n")
+        if os.path.isfile(checkpoint):
```

Comment:
this is not what this code is for. This 'resume' argument is meant to continue training a model for which training has stopped in between for some reason. It's not for loading the Roberta model to use in the GNN. `model = DocumentClassifier.load_from_checkpoint(checkpoint)` also won't work in this case because the document_classifier is not the roberta model. The roberta model in the roberta_finetuned_gnn is only used for creating the document embeddings. It is not the classifier itself that you load here.

Reply:
We could leave out the resuming part for further training a model for which training has been stopped earlier. But then you should just delete this code as it is not meant for loading the roberta model to use for document embeddings

Reply:
Okay, yeah then I will delete it. I did not imagine this situation, where you want to stop the training and then continue - but now I get what you mean. Well are you actively using it? If so we should leave it... (but adapt it)